### PR TITLE
[Oxfordshire] Send RDI reports by email.

### PR DESCRIPTION
**Pull request by dracos**
This factors out the RDI reporting, and adds a script to send RDI files every day, for each inspector.
For https://github.com/mysociety/fixmystreetforcouncils/issues/183

### CODE REVIEW

File: bin/oxfordshire/send-rdi-emails [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
```
Comment (pezholio) [inline bin/oxfordshire/send-rdi-emails]: Could this code be factored out into a package in perllib/FixMyStreet/scripts? Would (maybe) make it easier to test too? Certainly would make it easier to reuse if we need to further down the line

File: bin/oxfordshire/send-rdi-emails [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+my $cobrand = FixMyStreet::Cobrand->get_class_for_moniker('oxfordshire')->new;
+my @inspectors = $cobrand->users->search({
+    'user_body_permissions.permission_type' => 'report_inspect'
+}, {
+    join => 'user_body_permissions',
+    distinct => 1,
+})->all;
+
+foreach my $inspector (@inspectors) {
```
Comment (dracos) [inline bin/oxfordshire/send-rdi-emails]: That's the spec: https://github.com/mysociety/fixmystreetforcouncils/issues/183#issuecomment-297329233

File: bin/oxfordshire/send-rdi-emails (hunk 1 above)
Comment (dracos) [inline bin/oxfordshire/send-rdi-emails]: I think I'd prefer leaving that for such a time as that might (hopefully never) happen ;) It's not a very long script, the main bulk is already factored out generating the RDI, other than that it's only a small loop.

File: bin/oxfordshire/send-rdi-emails (hunk 2 above)
Comment (pezholio) [inline bin/oxfordshire/send-rdi-emails]: Oh yeah, `one email per inspector`. Carry on then 😄

File: bin/oxfordshire/send-rdi-emails (hunk 1 above)
Comment (pezholio) [inline bin/oxfordshire/send-rdi-emails]: Fair enough 👍

**Comment by dracos** [comment]
Merged as 61eb613dd

**Pull request closed without merging**